src/Components/Paginator/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+import styles from './style.module.scss';
+
+function Paginator(props) {
+  // Need to pass in number of entries and desired entries per page
+  const { entries, entriesPerPage } = props;
+  const pages = Math.ceil(entries / entriesPerPage);
+  const pageNumbers = [];
+  const [currentPage, setCurrentPage] = useState(1);
+
+  // Below 3 lines can be reused on the page component
+  // const indexOfLastEntry = currentPage * entriesPerPage;
+  // const indexOfFirstEntry = indexOfLastEntry - entriesPerPage;
+  // const currentEntries = entries.slice(indexOfFirstEntry, indexOfLastEntry);
+
+  // Get pages
+  for (let i = 1; i <= Math.ceil(pages); i += 1) {
+    pageNumbers.push(i);
+  }
+
+  // Go to specific page
+  const paginate = (pageNumber) => setCurrentPage(pageNumber);
+
+  // Go to prior page
+  const priorPage = (pageNumber) => setCurrentPage(pageNumber - 1);
+
+  // Go to next page
+  const nextPage = (pageNumber) => setCurrentPage(pageNumber + 1);
+
+  return (
+    <Pagination aria-label="Page navigation example" className={styles.paginationStyle}>
+      {currentPage !== 1
+        && (
```

Comment:
please change this to: `<PaginationItem disabled={currentPage !== 1} ...`
we dont want to hide the element when you cannot navigate back, we just want to disable it

Reply:
Chris was saying we should have it disappear instead of disabling it. I'm open to either way.

Reply:
@tackc it's a bit of a jarring user experience to have these buttons jump on and off of the page. I think best practice is to just disable/grey-out un-useable pagination buttons. Might affect the layout too if these buttons are hiding/showing like this? let me know your thoughts.

Reply:
I see what @renburnett is talking about. It would probably be ideal to have the button always be there and have it disabled if you are on the first page. @brendobrendo, will you please update

Reply:
Will do!

Reply:
✅